Dokument zamawiającego, wersja finalna + wersje poprzednie - Kamil

kamilsajdak96 opened the pull request:
Sporządzenie dokumentu zamawiającego i wykonawcy.
Issues: #1 #2

kamilsajdak96 committed:
Add files via upload

Test gita

kamilsajdak96 committed:
Dokument zamawiajacego oraz wykonawcy

nels3 committed:
Przeniesienie dokumentu zamawiającego do szablonu

nels3 commented:
Przeniosłam dokument zamawiającego do templatki. Proszę o sprawdzenie pliku i dodanie brakujących miejsc. Posługiwać jako wzorem można się pliku: materialy_pomocnicze/virtual/Projekt/przyklad_pracy-moga_byc_bledy

nels3 committed:
Merge pull request #30 from kamilsajdak96/master

Merge master do branch Kamil - dodanie materiałów_pomocniczych

kamilsajdak96 committed:
Wersja 1.2

Dodanie nazwy zamawiającego, poprawka informacji dotyczących obywatelstwa oraz pensji w punkcie 3, wnioski, dodanie terminu

nels3 committed:
Wersja 1.3 - wymagane dzialanie

nels3 commented:
Nie mogę znaleźć tej informacji o identyfikatorze jeszcze. Albo jest dobrze ukryta, albo trzeba dodać :) Nie widzę także tej wersji z obywatelstwem.

nels3 commented:
Prośba jeszcze o usunięcie drugiego razu "Ubezpieczenie na życie", nie wyhaczyłam tego podczas sprawdzania.

nels3 commented:
I nie widzę tej wersji uzgodnionej z obywatelstwem.

nels3 committed:
Usuniecie powtorzenia

kamilsajdak96 committed:
Wersja 1.4

Poprawienie informacji dotyczących obywatelstwa oraz dodanie identyfikatora pracownika

kamilsajdak96 committed:
Wersja 1.4

nels3 commented:
Wersja dokumentu zamawiającego już dobra. Stworzony folder etap_pierwszy_final na przechowywanie wersji finalnych dokumentów.

nels3 committed:
Wersja 2.0f - finalna wersja dokumentu zamawiajacego

nels3 reviewed (approved):
Sprawdzona ostateczna wersja dokumentu zamawiającego.
